lib/processors/mentalhealthprocessor.py

```diff
@@ -0,0 +1,139 @@
+from tornado import gen
+import traceback
+import ujson as json
+
+from ..config import CONFIG
+from .lib.utils import process_api_handler
+from .lib.baseprocessor import BaseProcessor
+
+
+class MentalHealthProcessor(BaseProcessor):
+    name = 'mental_health'
+    limit = 200
+    keywords = ["concerned", "important", "love", "confused", "boredom", "greed", "amazing", "enjoyable", "terrible", "angry", "corrupt", "fun", "happiness", "comfort", "shitty", "nothing", "sharing", "privileged", "lucky", "grateful", "difficult", "impossible", "unfair", "exciting", "better", "worse", "upsetting", "easy", "boredom", "confusion", "pride", "confidence", "doubt", "fear", "disappointment", "diligence", "excitement", "sad", "interesting", "sexy", "love", "hate", "anxiety", "anger", "annoyance", "frustration", "empathy", "sympathy", "apathy", "concern", "sadness", "confusion", "condemnation", "anxiety", "enjoyment", "graciousness", "love", "hate", "impatience", "patience"]
```

Comment:
Could you break these keywords out into a file (maybe `mentalhealth.txt` either in the processors directory or in the `data/` directory) and load it on init?  You can have the init simply be:

```
def __init__(self):
    with open("data/mentalhealth.txt") as fd:
        self.keywords = [kw.strip() for kw in fd]
    super().__init__()
```